templates/static/country_names_alternate.csv

```diff
@@ -1,38 +1,38 @@
-Name,Code
-Russia,RU
-Czechia,CZ
-Taiwan,TW
-Iran,IR
-South Korea,KR
-Vietnam,VN
-Venezuela,VE
-Republic of Lithuania,LT
-Bolivia,CO
-Ivory Coast,CI
-Republic of Moldova,MD
-Tanzania,TZ
-Hashemite Kingdom of Jordan,JO
-Macedonia,MK
-Syria,SY
-Palestine,PS
-Brunei,BN
-Laos,LA
-Republic of the Congo,CG
-U.S. Virgin Islands,VI
-Cabo Verde,CV
-Eswatini,SZ
-Åland,AX
-Sint Maarten,SX
-Democratic Republic of Timor-Leste,TL
-St Kitts and Nevis,KN
-Kosovo,XK
-"Bonaire, Sint Eustatius, and Saba",BQ
-British Virgin Islands,VG
-Federated States of Micronesia,FM
-Saint Martin,MF
-São Tomé and Príncipe,ST
-Vatican City,VA
-Falkland Islands,FK
-North Korea,KP
-Cocos [Keeling] Islands,CC
-Saint Helena,SH

```

Comment:
Why does this show up as deletions? It looks like the file contents are identical save for the header line.

Reply:
I have no idea.  My local history only shows a change to the first line

Reply:
🤷‍♂ I can live with it.